Cargo.toml

```diff
@@ -20,3 +20,8 @@ serde_derive = "~1.0.3"
 default = ["std"]
 
 std = ["serde/std"]
+
+serde-collections = ["serde/collections"]
```

Comment:
Does it make sense to have all these different features? What about just having one `nightly` feature that does all of those things?